components/Form/index.js

```diff
@@ -1,108 +1,89 @@
     });
-    
-    router.query.data=data;
-  <EntryListHome data={data}/>
-    
-    setCity(cityname?.output)
+    setCity(cityname?.output);
     event.target.reset();
   }
-  return ( 
-<>
-  <StyledForm  onSubmit={handleSubmit}>
-    <label htmlFor="destination">Your Destination:</label>
-    <StyledInput
-      name="destination"
-      pattern="^[^\sa0-9].*$"
-      required
-      id="destination"
-      type="text"
-      value={city}
+  return (
+    <>
+      <StyledForm onSubmit={handleSubmit}>
+        <label htmlFor="destination">Your Destination:</label>
+        <StyledInput
+          name="destination"
+          pattern="^[^\sa0-9].*$"
+          required
+          id="destination"
+          type="text"
+          value={city}
           onChange={() => setCity(cityname?.output)}
-    />
-    <StyledDate> 
-    <label htmlFor="from">START</label>
-      <input
-        id="from"
-        name="from"
-        type="date"
-      />
-      <label htmlFor="to">END</label>
-      <input
-        id="to"
-        name="to"
-        type="date"
-      />
-      </StyledDate>
-      <label htmlFor="entryTitle">Entry Title:</label>
-      <input
-        id="entryTitle"
-        name="entryTitle"
-        type="text"
-        maxLength="33"
-        pattern="^[^\sa0-9].*$"
-        required
-      />
-      <label htmlFor="entry">Journal Entry:</label>
-      <StyledInput
-      id="entry"
-        rows="5"
-        type="text"
-        name="entry"
-        maxLength="200"
-        pattern="^[^\sa0-9].*$"
-        required
-      />
-      <button type="submit">save entry</button>
-    </StyledForm>
-</>
+        />
+        <StyledDate>
+          <label htmlFor="from">from</label>
+          <input id="from" name="from" type="date" />
+          <label htmlFor="to">to</label>
+          <input id="to" name="to" type="date" />
+        </StyledDate>
+        <label htmlFor="entryTitle">Entry Title:</label>
+        <input
+          id="entryTitle"
+          name="entryTitle"
+          type="text"
+          maxLength="35"
+          pattern="^[^\sa0-9].*$"
+          required
+        />
+        <label htmlFor="entry">Journal Entry:</label>
+        <StyledInput
+          id="entry"
+          rows="10"
+          type="text"
+          name="entry"
+          maxLength="200"
+          pattern="^[^\sa0-9].*$"
+          required
+        />
+        <button type="submit">save entry</button>
+      </StyledForm>
+    </>
   );
 }
 const StyledInput = styled.input`
-padding: 1px;
-margin: 1px;
-font-size: 1em;
-font-weight: bold;
-text-decoration: none;
-background-color: lightblue;
-
+  padding: 1px;
+  margin: 1px;
+  font-size: 1em;
+  background-color: lightblue;
 `;
-const StyledForm = styled.form`
-  display: grid;
-  top: 50px;
-  border: 2px solid black;
-  border-radius: 2rem;
-  gap: 0,5rem;
-  padding: 10px;
-  margin: 10px;
+const StyledDate = styled.div`
+  display: flex;
+  flex-direction: oooooo;
```

Comment:
flex-direction should be "column" or "row" (row is the default, so you might not need to put it)

Reply:
jap, you are right. i removed it, thx :)